### DISCUSSION TITLE
UI, Other misc

### DISCUSSION THREAD

**Discussion by omodia** [mienaiyami/yomikiru#132] [Ideas]

**Comment by omodia** [comment]
- This is another trivial detail, but for the sake of looks I am mentioning it.
- Because, the parenthesis should have gap before starting and after ending, when there is a word before or after it.
- Scroll Speed(with keys) -> Scroll Speed (with keys)
- Add(Epub) -> Add (EPub), 
- Remove(Epub) -> Remove (EPub)
- So on and so forth.

**Comment by mienaiyami** [comment]
Thanks for mentioning.

**Comment by omodia** [comment]
- This red focus words are unnecessary as there is no input box.
- To use previously defined color, go to hex and type #RRGGBBAA.

![1](https://user-images.githubusercontent.com/116664905/236809177-07936047-830a-4d62-ab1b-fd2c96aba038.png)

**Comment by omodia** [comment]
The new UI looks great.
You made the right choice by removing this in location column.

![image](https://github.com/mienaiyami/yomikiru/assets/116664905/6aaf9d73-82a8-436a-9e0f-678e92def3b0)

I did not think of that. Good idea.

**Reply by mienaiyami** [reply]
It was useful when there was no "Open in Reader" button and only "Open" button on top.
Now its visible only when cursor is over this "Open" button for those who might still want it.

**Comment by omodia** [comment]


**Reply by mienaiyami** [reply]
I dont think its possible to unlock it now because it was converted to discussion. If you want, you can delete the issue or ask me to delete it if you are unable to.

**Reply by omodia** [reply]
Please delete the issue as I am unable to delete it.

**Reply by mienaiyami** [reply]
Thanks, I'll change them soon.
